environment.yml

```diff
@@ -1,10 +1,14 @@
 name: pancancer-classifier
+channels:
+- r
+- bioconda
 dependencies:
+- r-checkpoint=0.4.0
 - jupyter=1.0.0
 - numpy=1.11.2
-- pandas=0.18.0
-- scikit-learn=0.18.0
+- pandas=0.19.2
```

Comment:
Consider upgrading pandas to `0.20`, which had some major new features but also some breaking changes. Only if you can easily test nothing breaks, but you'll want to be using pandas 0.20+ going forward.